aksel.nav.no/website/pages/eksempler/actionmenu/sub-menu.tsx

```diff
@@ -0,0 +1,139 @@
+                                Under behandling
+                              </ActionMenu.Item>
+                              <ActionMenu.Item onSelect={console.info}>
+                                Under kontroll
+                              </ActionMenu.Item>
+                            </ActionMenu.SubContent>
+                          </ActionMenu.Sub>
+                        </ActionMenu.SubContent>
+                      </ActionMenu.Sub>
+                      <ActionMenu.Sub>
+                        <ActionMenu.SubTrigger>
+                          Tildel saksbehandler
+                        </ActionMenu.SubTrigger>
+                        <ActionMenu.SubContent>
+                          <ActionMenu.Group label="Saksbehandlere">
+                            <ActionMenu.Item onSelect={console.info}>
+                              Ola Normann
+                            </ActionMenu.Item>
+                            <ActionMenu.Item onSelect={console.info}>
+                              Bo Ramberg
+                            </ActionMenu.Item>
+                            <ActionMenu.Item onSelect={console.info} disabled>
+                              Ole Olsen
+                            </ActionMenu.Item>
+                            <ActionMenu.Item onSelect={console.info} disabled>
+                              Janne Nilssen
+                            </ActionMenu.Item>
+                            <ActionMenu.Item onSelect={console.info}>
+                              Karin Jakobsen
+                            </ActionMenu.Item>
+                            <ActionMenu.Item onSelect={console.info}>
+                              Kari Nordmann
+                            </ActionMenu.Item>
+                          </ActionMenu.Group>
+                        </ActionMenu.SubContent>
+                      </ActionMenu.Sub>
+
+                      <ActionMenu.Divider />
+
+                      <ActionMenu.Item variant="danger" onSelect={console.info}>
+                        Slett sak
+                      </ActionMenu.Item>
+                    </ActionMenu.Group>
+                  </ActionMenu.Content>
+                </ActionMenu>
+              </Table.DataCell>
+            </Table.Row>
+          );
+        })}
+      </Table.Body>
+    </Table>
+  );
+};
+
+const data = [
+  {
+    id: "03121",
+    status: "Avslått",
+  },
+  {
+    id: "16066",
+    status: "Mottatt",
+  },
+  {
+    id: "18124",
+    status: "Godkjent",
+  },
+  {
+    id: "24082",
+    status: "Mottatt",
+  },
+];
+
+// EXAMPLES DO NOT INCLUDE CONTENT BELOW THIS LINE
+export default withDsExample(Example);
+
+/* Storybook story */
+export const Demo = {
+  render: Example,
+};
+
+export const args = {
+  index: 4,
+  desc: "Undermenyer lar deg forenkle komplekse grensesnitt og filter ved å flytte innholdet til en godt strukturert meny. Vi anbefaler på det meste to nivåer med undermenyer.",
```

Comment:
```suggestion
  desc: "Undermenyer lar deg forenkle komplekse grensesnitt og filter ved å flytte innholdet til en godt strukturert meny. Vi anbefaler maks to nivåer med undermenyer.",
```
I think "maks" is clearer here, or it _might_ be confused with "in most cases".